src/components/Autocomplete/components/MappedOption/MappedOption.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {memo} from 'react';
```

Comment:
We create a new option component to map the differences between the old + new API